### PR TITLE
Add update quorums operation to node plugin

### PR DESCRIPTION
**Pull request by wmb-software-consulting**
## Why are these changes needed?

Add update quorums operation to node plugin

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [x] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: node/plugin/tests/plugin_test.go
Comment (jianoaix) [inline node/plugin/tests/plugin_test.go:1]: Nice, thanks for adding tests!

File: node/plugin/tests/plugin_test.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+	transactor, err := eth.NewTransactor(logger, client, operator.NODE_BLS_OPERATOR_STATE_RETRIVER, operator.NODE_EIGENDA_SERVICE_MANAGER)
+	assert.NoError(t, err)
+	assert.NotNil(t, transactor)
+
+	return transactor
+}
+
+// Returns the decrypted ECDSA private key from the given file.
+func getECDSAPrivateKey(keyFile string, password string) (*keystore.Key, *string, error) {
```
Comment (jianoaix) [inline node/plugin/tests/plugin_test.go:258]: Is this the same as this? https://github.com/Layr-Labs/eigenda/blob/d83442aef2c5663dd830bdfbf562d0ff018753fb/node/plugin/cmd/main.go#L23
If so, we can pull it out and make it sharable in tests

File: node/plugin/cmd/main.go [hunk 2]
```diff
@@ -135,6 +135,14 @@ func pluginOps(ctx *cli.Context) {
 			return
 		}
 		log.Printf("Info: successfully opt-out the EigenDA, for operator ID: %x, operator address: %x", operatorID, sk.Address)
+	} else if config.Operation == "update-quorums" {
+		log.Printf("Info: Operator with Operator Address: %x is updating its quorums: %v", sk.Address, config.QuorumIDList)
+		err = node.UpdateOperatorQuorums(context.Background(), operator, tx, config.ChurnerUrl, true, logger)
+		if err != nil {
+			log.Printf("Error: failed to update quorums for operator ID: %x, operator address: %x, error: %v", operatorID, sk.Address, err)
```
Comment (jianoaix) [inline node/plugin/cmd/main.go:142]: Can we log the quorum IDs as well?

**Review by jianoaix — APPROVED** [review]
LGTM, have you tested beyond unit test?

**Comment by wmb-software-consulting** [comment]
> LGTM, have you tested beyond unit test?

Actually is being tested by integration tests

**Pull request merged**